maths/line_length.py

```diff
@@ -0,0 +1,62 @@
+from typing import Callable, Union
+import math as m
+
+def line_length(fnc: Callable[[Union[int, float]], Union[int, float]],
+                x_start: Union[int, float],
+                x_end: Union[int, float],
+                steps: int = 100) -> float:
```

Comment:
This is returning __str__, not __float__.

Reply:
My function should return a float. In the doctests, I've converted the return value into a string to allow formatting. I've kept it as float for now, let me know if that's ok.